android/filament-android/libfilament-jni.map

```diff
@@ -1,4 +1,4 @@
 LIBFILAMENT {
-  global: Java_com_google_android_filament_*; JNI*;
+  global: *filament*; JNI*;
```

Comment:
How much does this increase the base filament library?

Reply:
I think this increased it by almost 100 KB, which was negated by the typo optimization that I found.

In the future, maybe we should consider splitting the filament SO into two parts (JNI and non-JNI), since the latter would be useful for pure C++ clients.